### PR TITLE
Feature/mdr 134

### PR DESCRIPTION
**Pull request by costezki**
logger implemented

### CODE REVIEW

**Comment by cosletmihai** [comment]
I'd recommend, instead of the branch name to use the issue name instead of the issue code (that can be read from the branch name)

File: eds4jinja2/__init__.py [hunk 1]
```diff
@@ -9,4 +9,15 @@
 
 # The format of the __version__ line is matched by a regex in setup.py and /docs/conf.py
 __version__ = "0.1.19"
-__date__ = "2020-08-14"
\ No newline at end of file
+__date__ = "2020-08-14"
+
+import logging
+
+# hard coding the log level
```
Comment (cosletmihai) [inline eds4jinja2/__init__.py:16]: what do you mean by "hard coding"?

File: eds4jinja2/adapters/base_data_source.py [hunk 2]
```diff
@@ -45,8 +49,11 @@ def fetch_tabular(self) -> Tuple[Optional[object], Optional[str]]:
             if not self._can_be_tabular():
                 raise UnsupportedRepresentation("Only TREE representation is supported")
             else:
-                return self._fetch_tabular(), None
+                result = self._fetch_tabular()
+                logger.info(f"With {type(self).__name__}, fetching as tabular - {str(self)}", extra={'class': self.__class__.__name__})
```
Comment (cosletmihai) [inline eds4jinja2/adapters/base_data_source.py:53]: it seems that 
```python
type(self).__name__
# and
self.__class__.__name__
```
achieve the same thing, why not use only one version?

File: eds4jinja2/__init__.py (hunk 1 above)
Comment (costezki) [reply eds4jinja2/__init__.py:16]: well, ideally this shall be in an .ini/.cfg file

File: eds4jinja2/adapters/base_data_source.py (hunk 2 above)
Comment (costezki) [reply eds4jinja2/adapters/base_data_source.py:53]: deal, that was a mistake

**Pull request merged**